### PR TITLE
UI changes to wallet initialization

### PR DESCRIPTION
**Pull request by Tyratox**
This PR changes the UI of the wallet initialization to make it more consistent with the rest of the application. (No negative text, no blue background, buttons in the toolbar)

Screenshot 2022-12-20 at 14 42 21
Screenshot 2022-12-20 at 14 44 04

### CODE REVIEW

File: fe1-web/src/resources/strings.ts [hunk 1]
```diff
@@ -397,15 +397,23 @@ namespace STRINGS {
     ', you can continue and start using the POPStellar application. ' +
     'The purpose of this application is to demonstrate use cases for proofs of personhood (PoP).';
 
-  export const wallet_welcome_start_exploring = 'Start exploring POPStellar 🤩';
+  export const wallet_welcome_already_know_seed_1 =
+    'If you have already used this application before and remember your previous';
+  export const wallet_welcome_already_know_seed_2 =
+    ', you can restore your wallet by pressing the respective button.';
```
Comment (pierluca) [inline fe1-web/src/resources/strings.ts:403]: does it make sense that we start with a comma ?

File: fe1-web/src/features/wallet/screens/WalletCreateSeed.tsx [hunk 2]
```diff
@@ -83,75 +77,72 @@ const WalletCreateSeed = () => {
...
-        <View style={containerStyles.centeredY}>
-          <Text style={[Typography.heading, Typography.negative]}>
-            {STRINGS.wallet_welcome_heading}
+    <ScreenWrapper toolbarItems={toolbarItems}>
+      <View style={containerStyles.centeredY}>
+        <Text style={Typography.heading}>{STRINGS.wallet_welcome_heading}</Text>
+        <Text style={Typography.paragraph}>{STRINGS.wallet_welcome_text_first_time}</Text>
+        <Text style={Typography.paragraph}>
+          <Text>{STRINGS.wallet_welcome_text_wallet_explanation_1}</Text>
+          <Text> </Text>
```
Comment (pierluca) [inline fe1-web/src/features/wallet/screens/WalletCreateSeed.tsx:100]: does it make sense to have all these empty `<Text>` fields ? 
Is there no better way ?

**Review by pierluca — COMMENTED** [review]
Minor comments, but LGTM overalls

File: fe1-web/src/resources/strings.ts (hunk 1 above)
Comment (Tyratox) [reply fe1-web/src/resources/strings.ts:403]: This sentence is split into two parts with "seed 🔑" in between (colored in blue). If we would have it the same color or put the comma inside a <Text/> as with the spaces, we would could combine the two / not start the sentence with a comma. What do you think?

Screenshot 2022-12-20 at 14 42 21

File: fe1-web/src/features/wallet/screens/WalletCreateSeed.tsx (hunk 2 above)
Comment (Tyratox) [reply fe1-web/src/features/wallet/screens/WalletCreateSeed.tsx:100]: Well either that or have sentences start or end with a space which is bad too 😅

Comment (Tyratox) [reply fe1-web/src/features/wallet/screens/WalletCreateSeed.tsx:100]: Or, of course, rewrite the sentences / combine them as suggested above. The reason for these artifacts are the differently colored text sections. Or do you see a another option?

**Review by pierluca — APPROVED** [review]
Okay, I understand better. We could do better, but there's no point in investing more effort in this now. Good enough ! 👍

**Pull request merged**